testsuites/basic/lsvmbus.py

```diff
@@ -0,0 +1,112 @@
+import math
+from pathlib import Path
+from typing import Optional
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.environment import EnvironmentStatus
+from lisa.features import SerialConsole
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lscpu, Lsvmbus
+from lisa.util import LisaException
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite uses to check correctness of vmbus channel
+    """,
+    tags=[],
+)
+class LsVmBus(TestSuite):
+    TIME_OUT = 300
+    __Tokens = [
+        "Operating system shutdown",
+        "Time Synchronization",
+        "Heartbeat",
+        "Data Exchange",
+        "mouse",
+        "keyboard",
+        "Synthetic network adapter",
+        "Synthetic SCSI Controller",
+    ]
+
+    @TestCaseMetadata(
+        description="""
+        This test is to check channel count of each netvsc and storvsc SCSI device.
+        When count is unexpected, then test fail.
+        """,
+        priority=0,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole],
+        ),
+    )
+    def lsvmbus_test(self, case_name: str) -> None:
+        node = self.environment.default_node
+        case_path: Optional[Path] = None
+
+        is_ready, tcp_error_code = wait_tcp_port_ready(
+            node.public_address, node.public_port, log=self.log, timeout=self.TIME_OUT
+        )
+        if not is_ready:
+            serial_console = node.features[SerialConsole]
+            case_path = self._create_case_log_path(case_name)
+            serial_console.check_panic(saved_path=case_path, stage="bootup")
+            raise LisaException(
+                f"cannot connect to [{node.public_address}:{node.public_port}], "
+                f"error code: {tcp_error_code}, no panic found in serial log"
+            )
+        node.is_connected = True
+        environment_information = node.get_node_information()
+        if environment_information["vm_generation"] == "1":
+            self.__Tokens.append("Synthetic IDE Controller")
+
+        lsvmbus_tool = node.tools[Lsvmbus]
+        vmbus_results = lsvmbus_tool.get_verbose_messages()
+        for token in self.__Tokens:
```

Comment:
1. If `__Tokens` is updated in code, it should be in `__init__` to be an instance variable.
2. As the test case is last level of subclass, the private members should be like `_tokens`, not `__tokens`